### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by chingor13**
Allows the extension to compile on windows.

* adds a `config.w32` file
* removes variable length array declarations
* uses php's internal windows implementation of the c function `gettimeofday`

### CODE REVIEW

File: config.w32 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// $Id$
+// vim:ft=javascript
+
+ARG_ENABLE("stackdriver-trace", "stackdriver_trace support", "no");
+
+if (PHP_STACKDRIVER_TRACE != "no") {
+    EXTENSION('stackdriver_trace', 'stackdriver_trace.c stackdriver_trace_context.c stackdriver_trace_span.c');
```
Comment (chingor13) [inline config.w32:7]: @stanley-cheung I think this is all this extension needs as the code is all in one place (in the root).

File: stackdriver_trace.c [hunk 2]
```diff
@@ -599,7 +604,8 @@ PHP_FUNCTION(stackdriver_trace_list)
     int i = 0;
     stackdriver_trace_span_t *trace_span;
     int num_spans = STACKDRIVER_TRACE_G(spans)->nNumUsed;
-    zval labels[num_spans], spans[num_spans];
+    zval *labels = emalloc(num_spans * sizeof(zval));
```
Comment (tmatsuo) [inline stackdriver_trace.c:607]: I'm not familiar with the PHP internal memory management, but is it safe to do:
emalloc -> array_init -> (and array_init in stackdriver_labels_to_zval_array) -> efree
?

Comment (chingor13) [reply stackdriver_trace.c:607]: Good point, I'm calling `array_init` twice. Each one allocs more memory, so we're leaking memory here. I'll address this in a separate PR.

Comment (tmatsuo) [reply stackdriver_trace.c:607]: Alright, then LGTM for the windows support

**Review by tmatsuo — APPROVED** [review]

File: stackdriver_trace.c (hunk 2 above)
Comment (chingor13) [reply stackdriver_trace.c:607]: Addressed in #7

**Pull request merged**